assets/queries/cloudFormation/vpc_has_FlowLog_disabled/query.rego

```diff
@@ -0,0 +1,34 @@
+package Cx
+
+CxPolicy [ result ] {
+  resource := input.document[i].Resources[name]
+  resource.Type = "AWS::EC2::VPC"
+    
+  not CheckFlowLogExistance(name)
+    
+  		
+	result := {
+                "documentId": 		input.document[i].id,
+                "searchKey": 	    sprintf("Resources.%s", [name]),
+                "issueType":		"IncorrectValue",  
+                "keyExpectedValue": sprintf("Resources.%s has FlowLogs enabled", [name]),
+                "keyActualValue": 	sprintf("Resources.%s has FlowLogs disabled", [name])
+              }
+}
+
+CheckFlowLogExistance (service) = result {
+    
+	documents := input.document[index].Resources[a]
+  documents.Type = "AWS::EC2::FlowLog"
+    
+  result := contains(documents, service)
```

Comment:
the contains function here could be deleted and we could return directly since it's a simple operation:
`result := documents[_].ResourceId == service`

Reply:
Done